app/controllers/users_controller.rb

```diff
@@ -17,14 +17,15 @@ def create
     end
   end
 
+
   def show
 
   end
 
   private
 
   def user_params
-    params.permit(:first_name, :last_name, :email, :password, :password_confirmation)
+    params.permit(:first_name, :last_name, :email, :password)
```

Comment:
Validating password_confirmation as a params was causing part of the issue.

Reply:
glad we were able to come up with a simple solve for this! I agree with Jack that asking Dione on Monday is a good idea to check and make sure this is ok in terms of security.

Reply:
indeed, nice work Sage